teams/community-localization.toml

```diff
@@ -0,0 +1,29 @@
+name = "community-localization"
+subteam-of = "community"
+
+[people]
+leads = ["vertexclique"]
+members = [
+    "Manishearth",
+    "JohnTitor",
+    "XAMPPRocky",
+    "vertexclique"
+]
+
+[rfcbot]
+label = "T-localization"
+name = "Localization"
+ping = "rust-lang/localization"
+
+[website]
+name = "Localization team"
+description = "Working on localization of compiler, documentation and websites"
+repo = "https://github.com/rust-lang/community-localization"
```

Comment:
I would like to change the repo at https://github.com/rust-lang/project-localisation to this name.
Since there will be inconsistencies between team data and the repo itself.